package.json

```diff
@@ -20,6 +20,7 @@
     "http-proxy-agent": "^5.0.0",
     "https-proxy-agent": "^5.0.0",
     "vscode-nls": "^5.0.0",
+    "vscode-jsonrpc": "^5.0.0",
```

Comment:
I'd rather not add any new dependencies. It's ok to duplicate the types and implementations.

Reply:
This is needed for unit testing, and this package is added to `devDependencies` and does not increase the size of the product